tests/behat/features/userforms.feature

```diff
@@ -54,7 +54,10 @@ Feature: Userforms
     # Click save on the file upload modal to use the default "Form-submissions" folder
     And I press the "Save and continue" button
     And I wait for 5 seconds
-    And I press the "Save" button
+    And I select "File Upload Field" from the "Form_Fields_GridFieldEditableColumns_7_ClassName" field
+    And I press the "Save and continue" button
```

Comment:
It's weird, but we have to select "File Upload Field" again, since it looks like Behat is processing all steps faster than actual value is saved.